src/main/java/by/yahor_kulesh/App.java

```diff
@@ -1,46 +1,37 @@
 package by.yahor_kulesh;
 
-import by.yahor_kulesh.model.Ticket;
-import by.yahor_kulesh.services.TicketService;
+import by.yahor_kulesh.model.Data;
+import by.yahor_kulesh.model.Printable;
+import by.yahor_kulesh.model.tickets.*;
+import by.yahor_kulesh.model.users.*;
 import by.yahor_kulesh.validators.InputValidator;
 
+import java.math.BigDecimal;
+import java.time.ZoneId;
 
-public class App {
+
+public class App extends Data {
```

Comment:
why did you remove all logic from TicketService and move it into new class App?

Reply:
In my opinion TicketService is used to store logic to work with Tickets only, not common methods and main. Main method needs to be stored in separate class which must be starting point of application. In that case i moved all this logic to App.
Now i moved it back to TicketService class and call it from App

Reply:
I resolved others problems too

Reply:
Yes, agree, but in this case, TicketService, if it presents into the app, has to do smth, it can't be empty. In your case you just moved all from TicketService to App, after that you moved all from App to TicketService. Looks like one of these classes is redundant here.